libs/core-scanner/src/pages/robots-txt.ts

```diff
@@ -70,14 +70,17 @@ const findRobotsCrawlDelay = (
   logger: Logger,
   logData: any,
   robotsTxt: string,
-) => {
+): number | null => {
   const directives = robotsTxt.split('\n');
-  let crawlDelay = -1;
+  let crawlDelay: number = null;
 
   for (const directive of directives) {
     if (directive.toLowerCase().startsWith('crawl-delay:')) {
       try {
         crawlDelay = parseInt(directive.split(' ')[1]);
+        if(isNaN(crawlDelay)){
+          crawlDelay = -1;
```

Comment:
Let's set it `null` here also.

```suggestion
          crawlDelay = null;
```